### ISSUE TITLE
Review the site for accessibility

### ISSUE THREAD

**Issue by mcataford** [tophat/opensource.tophat.com#63]
The website gets dinged pretty heavily by aXe, we should at the very least make sure that there are no violations there. It would also be nice to ensure that the website is usable via keyboard only.

Any other suggestions welcome!

**Comment by ju-nc** [comment]
aXe was really useful in flagging a lot of our semantic issues. The only significant thing left for this is to make the project list an HTML list, which makes a lot more sense layout-wise for keyboard/screenreader visitors.

There are a few small issues left:

- Text is overlaid on a non-solid background
- When navigating using a screenreader, the "stars" section of the project information is a group and takes multiple keys to get out of

Colour contrast meets AA at least, the pink doesn't meet 5:1 contrast but it should be fine since it's not used for text that is super small.
